ci/docker/Dockerfile.maldet.leap.15

```diff
@@ -0,0 +1,132 @@
+# Copyright 2018-2022 Intel Corporation
```

Comment:
Same concerns in this file as for the `el.8` one.

Reply:
Should be fixed in current push.  All common code between the Dockerfiles used for DAOS and for malware scanning have been moved to scripts.